unix-opts.lisp

```diff
@@ -591,7 +592,7 @@ The output goes to STREAM."
                               2) ; colon and space
                            defined-options)
               args))
-    (when defined-options
+    (when (and (or (not usage-of) (not brief)) defined-options)
```

Comment:
I think it is a bit clearer if we would change this to `(and (not (and usage-of brief)) defined-options)` which is the same according to the De Morgan's law.